plutus-core/untyped-plutus-core/test/Evaluation/Builtins/Bitwise1.hs

```diff
@@ -2,7 +2,8 @@
 {-# LANGUAGE OverloadedStrings #-}
 {-# LANGUAGE TypeApplications  #-}
 
-module Evaluation.Builtins.Laws (
+-- | Tests for [CIP-0122](https://cips.cardano.org/cip/CIP-0122).
+module Evaluation.Builtins.Bitwise1 (
```

Comment:
Wouldn't it be more self-descriptive to name it `Evaluation.Builtins.Bitwise.CIP0122` and `Evaluation.Builtins.Bitwise.CIP0123` ?

Reply:
OK, I'll do that.  I thought about it and couldn't make up my mind, but you've persuaded me now.  I alos thought about putting them all in one file, but they're both pretty big already.